src/lib/packages/helpers.ts

```diff
@@ -402,7 +402,18 @@ export const actOnTailwindGenerator = (
       )} text-transparent bg-clip-text`;
       break;
     case 'gradient-border':
-      codeToCopy = ``;
+      element = outputElement.style;
+      const content = window.getComputedStyle(outputElement, '::before');
```

Comment:
this caused an error because you aren't using it:
![image](https://github.com/Dun-sin/Code-Magic/assets/78784850/568afc13-cde4-44e1-ab00-602ff0dd7781)